packages/server/lib/open_project.ts

```diff
@@ -238,6 +238,20 @@ export class OpenProject {
     this.projectBase.server._socket.changeToUrl(newSpecUrl)
   }
 
+  changeUrlToDebug () {
```

Comment:
I think we could make this API function a bit more useful by accepting a param of what route slug to move to - we may need a way to navigate user to Settings or other areas of the app in the future

```ts
type Route = '/specs' | '/specs/runner' | '/debug' | '/runs' | '/settings'
moveTo (route: Route) {
...
}
```

....or make it totally open and just expose a `goToUrl` function that takes a string so line 238 could use it, but you might be able to break the app by going to a bad url 🤔

Reply:
Interested in other reviewers' thoughts here - worth making this more flexible? Should it be exposed upwards through `ProjectActions` even though it may not really be a "Project" action to move to "Settings" or another page?

Reply:
In the short term I updated to accepting the pre-defined route, matching the above code example, but I'm certainly open to thoughts on the `goToUrl`.

Reply:
Based on feedback, reverting to narrow implementation until we have broad usage information for the API.